python/cuml/cluster/hdbscan.pyx

```diff
@@ -397,6 +423,12 @@ class HDBSCAN(Base, ClusterMixin, CMajorInputTagMixin):
         persist the clustering object for later re-use you probably want
         to set this to True.
 
+    prediction_data : bool, optinal (default=False)
```

Comment:
If you are not able to get to exposing the actual `prediction_data_` attribute using hdbscan's `PredictionData` Python object, can you create a Github issue for this please so it doesn't get forgotten?